### PR TITLE
feat(template-search): add template - INNO-1315

### PR DESCRIPTION
**Pull request by emeryro**
# PR description

Add template according to [specifications](https://webgate.ec.europa.eu/CITnet/confluence/display/NEXTEUROPA/Search+page+template+-+Specs)

### CODE REVIEW

**Comment by HAL-Patch-INNO** [comment]
Deploy preview for *europa-component-library* ready!

Built with commit 90c82b47768a089b1713a4e02c7a41447f69bcb7

https://deploy-preview-994--europa-component-library.netlify.com

File: src/systems/ec/implementations/react/packages/button/Button.jsx [hunk 1]
```diff
@@ -6,6 +6,7 @@ import Icon from '@ecl/ec-react-component-icon/Icon';
 const Button = ({
   variant,
   type,
+  block,
```
Comment (degliwe) [inline src/systems/ec/implementations/react/packages/button/Button.jsx]:
```suggestion
```

File: src/systems/ec/implementations/react/packages/button/Button.jsx [hunk 2]
```diff
@@ -15,6 +16,7 @@ const Button = ({
 }) => {
   const classNames = classnames(className, 'ecl-button', {
     [`ecl-button--${variant}`]: variant,
+    [`ecl-button--block`]: block,
```
Comment (degliwe) [inline src/systems/ec/implementations/react/packages/button/Button.jsx]:
```suggestion
```

File: src/systems/ec/implementations/react/packages/button/Button.jsx [hunk 3]
```diff
@@ -47,6 +49,7 @@ const Button = ({
 Button.propTypes = {
   variant: PropTypes.string,
   type: PropTypes.string,
+  block: PropTypes.bool,
```
Comment (degliwe) [inline src/systems/ec/implementations/react/packages/button/Button.jsx]:
```suggestion
```

File: src/systems/ec/implementations/react/packages/button/Button.jsx [hunk 4]
```diff
@@ -57,6 +60,7 @@ Button.propTypes = {
 Button.defaultProps = {
   variant: 'primary',
   type: 'submit',
+  block: false,
```
Comment (degliwe) [inline src/systems/ec/implementations/react/packages/button/Button.jsx]:
```suggestion
```

**Review by degliwe — APPROVED** [review]
This is the best we can do without extra specs from design.

**Pull request merged**